molecule/molecule.h

```diff
@@ -214,6 +214,7 @@ namespace indigo
 
             Array<_AttachOrder> order; // attach order info
         };
+        
```

Comment:
There is no essential change in this file. It's worse to exclude from the whole file from pr.